src/components/DeleteCompletedButton/DeleteCompletedButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  onDeleteCompletedTodos: () => void,
+};
+
+export const DeleteCompletedButton: React.FC<Props> = ({
+  todos,
+  onDeleteCompletedTodos,
+}) => {
+  const completedTodos = todos.filter(todo => todo.completed).length;
+
+  return (
+    <>
+      {completedTodos ? (
+        <button
+          type="button"
+          className="todoapp__clear-completed"
+          onClick={onDeleteCompletedTodos}
+        >
+          Clear completed
+        </button>
+      ) : (
+        <button
```

Comment:
Consider using conditional rendering for only the attributes that change, instead of duplicating the entire button element.

Reply:
![Screenshot 2023-06-18 at 21 10 31](https://github.com/mate-academy/react_todo-app/assets/81584108/e0eb56b1-04a7-4780-b969-5682c91aa413)

Is there a way to make it more elegant?